lib/tags/group-creator.tag

```diff
@@ -128,8 +128,10 @@
                         if(response.success) {
                             self.refs.title.classList.remove('graphjs-error');
                             self.refs.title.classList.add('graphjs-success');
+                            self.refs.title.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.title refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
i've added data-attribute where ever there is `content` in css, we can remove that line but
as pseudo-elements are not supported for input we may have to remove all css in all .less files right?? which is not related to this PR.. should just remove this line for now?